extensions/binding/org.eclipse.smarthome.binding.homematic/src/main/java/org/eclipse/smarthome/binding/homematic/internal/communicator/AbstractHomematicGateway.java

```diff
@@ -518,6 +518,11 @@ protected void setChannelDatapointValues(HmChannel channel, HmParamsetType param
         }
     }
 
+    @Override
+    public void loadDatapointValue(HmDatapoint dp) throws IOException {
+        getRpcClient(dp.getChannel().getDevice().getHmInterface()).getDatapointValue(dp);
```

Comment:
is this null save? looks like a lot of calls wo any check.

Reply:
Conceptually, it should be save. A datapoint will always belong to a channel, which always belongs to a device, which always is connected to an interface of the homematic gateway. Moreover, there is one other place in this class where a very similar call is already in use (and does not seem to produce any NPEs):
```
private void sendDatapoint(final HmDatapoint dp, final HmDatapointConfig dpConfig, final Object newValue,
        final boolean ignoreVirtualDatapoints) throws IOException, HomematicClientException {
                    ...
line 682            getRpcClient(dp.getChannel().getDevice().getHmInterface()).setDatapointValue(dp, newValue);
                    ...
}
```

If you would rather have null checks anyway to be safe, then I can of course add them, but I don't think that they are necessary in this case.